### PR TITLE
docs: 7.8.0 release notes, breaking changes, highlights

### PR DESCRIPTION
**Pull request by bmorelli25**
##  Motivation/summary

* Adds release notes, breaking changes, and highlights for `7.8.0`.
  * No breaking changes for `7.8.0`
  * Highlights include service maps, deployment annotations, RUM + Maps, RUM Payload compression, and .NET SqlClient support.
* Moves release notes for each minor to one page. This reduces the number of clicks needed to view release notes and makes it easier to scan minor releases for changes.

## Related issues

* 7.8 changes are for https://github.com/elastic/observability-dev/issues/833.
* Moving release notes to one page closes https://github.com/elastic/apm-server/issues/3324.

## Doc preview

[Preview link](https://apm-server_3869.docs-preview.app.elstc.co/diff)

## Redirects needed

Redirects for the following pages will be activated in a separate PR when `7.8` is released:

```
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.7.1.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.7.0.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.6.2.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.6.1.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.6.0.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.5.2.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.5.1.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.5.0.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.4.1.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.4.0.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.3.2.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.3.1.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.3.0.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.2.1.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.2.0.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.1.1.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.1.0.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.0.1.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.0.0.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.0.0-rc2.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.0.0-rc1.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.0.0-beta1.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.0.0-alpha2.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-7.0.0-alpha1.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.8.10.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.8.9.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.8.8.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.8.7.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.8.6.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.8.5.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.8.4.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.8.3.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.8.2.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.8.1.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.8.0.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.7.2.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.7.1.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.7.0.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.6.2.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.6.1.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.6.0.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.5.4.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.5.3.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.5.2.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.5.1.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.5.0.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.4.3.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.4.2.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.4.1.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.4.0.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.3.2.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.3.1.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.3.0.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.2.4.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.2.3.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.2.2.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.2.1.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.2.0.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.1.4.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.1.3.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.1.2.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.1.1.html
https://www.elastic.co/guide/en/apm/server/current/release-notes-6.1.0.html


```

### CODE REVIEW

**Comment by bmorelli25** [comment]
I'm going to merge this so I can unblock the stack release notes. If anyone has any feedback, feel free to leave a review on this PR and I'll update in a new PR.

File: changelogs/7.8.asciidoc [hunk 1]
```diff
@@ -5,29 +5,27 @@ https://github.com/elastic/apm-server/compare/7.7\...7.8[View commits]
...
+* Add cloud metadata intake {pull}3729[3729].
 
 [float]
 ==== Added
-* RUM V3 endpoint {pull}3328[3328], {pull}3414[3414], {pull}3512[3512], {pull}3526[3526], {pull}3616[3616], {pull}3648[3648], {pull}3659[3659], {pull}3733[3733]
-* We now publish libbeat metrics through the /debug/vars endpoint {pull}3550[3550]
-* Introduce transaction.duration.histogram aggregation {pull}3651[3651]
-* Introduce config to drop unsampled transactions {pull}3702[3702]
+* RUM V3 endpoint {pull}3328[3328], {pull}3414[3414], {pull}3512[3512], {pull}3526[3526], {pull}3616[3616], {pull}3648[3648], {pull}3659[3659], {pull}3733[3733].
+* We now publish libbeat metrics through the /debug/vars endpoint {pull}3550[3550].
```
Comment (graphaelli) [inline changelogs/7.8.asciidoc:27]:
```suggestion
* Publish libbeat metrics through the /debug/vars endpoint {pull}3550[3550].
```

File: docs/guide/apm-release-notes.asciidoc [hunk 2]
```diff
@@ -28,6 +29,74 @@ For a full list of changes, see the
 
 // end::notable-v8-highlights[]
 
+[[release-highlights-7.8.0]]
+=== APM version 7.8.0
+
+// tag::notable-v78-highlights[]
+Service maps::
+
+Service maps now integrate with machine learning—​offering real-time health indicators, based
```
Comment (graphaelli) [inline docs/guide/apm-release-notes.asciidoc:38]:
```suggestion
Service maps now integrate with machine learning, offering real-time health indicators based
```

**Comment by cachedout** [comment]
This change has caused a test to fail:

https://apm-ci.elastic.co/job/apm-server/job/apm-server-check-changelogs-mbp/job/master/

The pipeline in question runs [this script](https://github.com/elastic/apm-server/blob/master/script/check_changelogs.py).

@bmorelli25 is this something you'd like to take a look at? If we need to update the check script, we can certainly do that as well. Let me know what you think. Thanks!

**Comment by bmorelli25** [comment]
@cachedout, we had a conversation about this on Slack yesterday. I'll Slack you the internal link. The consensus was to remove this check and only run the script manually.

**Pull request merged**